src/magicbox/SuggestionsManager.ts

```diff
@@ -94,6 +94,9 @@ export class SuggestionsManager {
     const selected = this.keyboardFocusedSuggestion;
     if (selected != null) {
```

Comment:
I think it is confusing to put this behaviour which mutates the state inside of a method with `get` in the name. If I call `getKeyboardFocusedElement` twice, I will get a result the first time but null the second time. I think a better name would be `selectAndReturnKeyboardFocusedElement`

Reply:
It's more getting, triggering & unassigning KeyboardFocusedElement

`selectAndReturnKeyboardFocusedElement` 😐